### PR TITLE
feature/2609-migration-leftover-tasks

### PR DESCRIPTION
**Pull request by MarcEgliP**

### CODE REVIEW

File: doc/development/04_wagons.md [hunk 1]
```diff
@@ -253,11 +253,14 @@ If wished, the attributes can be included in the CSV-File that is generated when
...
-      indexes title
-    end
+To make new attributes searchable you have to include them in the searchable attributes
+of the corresponding model:
+
+    SEARCHABLE_ATTRS = [
+    :first_name, :last_name, :company_name, :nickname, :email, :address, :zip_code, :town,
+    :country, :birthday, :additional_information, {phone_numbers: [:number],
+                                                   social_accounts: [:name], additional_emails: [:email]}
+]
```
Comment (njaeggi) [inline doc/development/04_wagons.md:263]: Also document the search_strategies domain classes and add some documentation that PgSesarchable has to be included in the corresponding model

Comment (njaeggi) [inline doc/development/04_wagons.md:263]: Please note that PGSearchable has to be initialzed again, also note that we have a search strategy for each class, for maybe special cases like person search but also for the basic searches to check min length for terms, we never use the model search scope directly, to improve performance.

**Review by njaeggi — CHANGES_REQUESTED** [review]
Just one small change to documentation, the rest looks good to me :)

File: doc/architecture/wagons/README.md [hunk 2]
```diff
@@ -271,6 +271,9 @@ of the corresponding model:
                                                    social_accounts: [:name], additional_emails: [:email]}
 ]
 
+Dont forget to include de PgSearchable module too:
```
Comment (njaeggi) [inline doc/architecture/wagons/README.md:274]: de?

**Review by njaeggi — CHANGES_REQUESTED** [review]

File: doc/architecture/wagons/README.md [hunk 3]
```diff
@@ -262,11 +262,17 @@ If wished, the attributes can be included in the CSV-File that is generated when
 
 ### Make attributes searchable
```
Comment (njaeggi) [inline doc/architecture/wagons/README.md:263]: This isn't really a wagon documentation, I would maybe move this somewhere else, and only explain how to add something to SEARCHABLE_ATTRS in wagons here :rocket:

**Review by njaeggi — CHANGES_REQUESTED** [review]

**Comment by lkleisa** [comment]
@MarcEgliP Nice work! 🚀

**Pull request merged**